project/build.scala

```diff
@@ -1,11 +1,12 @@
 import sbt._
 
+
 object Build extends sbt.Build {
-  lazy val geospace= Project( "geospace-proj", file(".") ).
-    settings(BuildSettings.buildSettings : _*).
+  lazy val geospace = Project( "geospace-root", file("."), settings = BuildSettings.rootSettings).
+    settings(BuildSettings.buildSettings: _*).
```

Comment:
If I am not mistaking, it will not override the original settings set to itself (the root project). The .settings will then be added on top and will be used for the aggregate projects. The .settings itself is not really needed however, imho, as the settings are added anyway in the project settings files of each sub-project. You can check that this pattern is followed in other projects also, but I would argue the need to do this.